assets/queries/ansible/aws/user_data_shell_script_is_encoded/test/positive_expected_result.json

```diff
@@ -0,0 +1,7 @@
+[
+	{
+		"queryName": "User Data Shell Script Is Encoded",
+		"severity": "HIGH",
+		"line": 1
```

Comment:
Despite this being correct with the engine, the line that should be identified should be 9, where `user_data` is present